### DISCUSSION TITLE
Questions about how to cancel a notice

### DISCUSSION THREAD

**Discussion by fernarr** [OP-TED/eForms-SDK#714] [Q&A] [answered]
Dear TED Members,

I have some questions about how to cancel a notice with the “Change notice”.

The documentation and technical workshop show us that it is possible to use the change notice to cancel a notice that has been published using `<cbc:ReasonCode languageID="EST" listName="change-corrig-justification">cancel</cbc:ReasonCode >`

The questions are:

- Is it necessary to send all the information about the notice that we want to cancel? For example, should we send the tender submission deadline? The problem we are seeing in this case is that all business rules apply, so if we want to cancel a notice with the tender submission deadline passed, it is impossible to cancel the notice (validation failed).
- Can we cancel a notice that was sent in the past with eSentool (TED XML format)? In this case, we do not have all the data that are now require with eForms.
- Is it possible to send a notice, correct it (update information) and finally cancel it?

We have some questions about how to cancel a notice with eForms and haven't found any examples on TED. Could you show us an example of this? We have seen the example at https://github.com/OP-TED/eForms-SDK/blob/develop/examples/notices/change-cn_24_void.xml, but it has the problems I explained before.

Thanks for all

**Comment by YvesJo** [comment, marked as answer]
Please get in touch with the helpdesk ted@publications.europa.eu
